BDArmory/Settings/BDArmorySettings.cs

```diff
@@ -42,18 +42,19 @@ public class BDArmorySettings
         //[BDAPersistentSettingsField] public static bool INSTAKILL = true; //Deprecated, only affects lasers; use an Instagib mutator isntead
         [BDAPersistentSettingsField] public static bool AI_TOOLBAR_BUTTON = true;                 // Show or hide the BDA AI toolbar button.
         [BDAPersistentSettingsField] public static bool VM_TOOLBAR_BUTTON = true;                 // Show or hide the BDA VM toolbar button.
-        [BDAPersistentSettingsField] public static bool INFINITE_AMMO = false;              //infinite Bullets/rockets/laserpower
-        [BDAPersistentSettingsField] public static bool INFINITE_ORDINANCE = false;         //infinite missiles/bombs (on ordinance w/ Reload Module)
-        [BDAPersistentSettingsField] public static bool LIMITED_ORDINANCE = false;         //MML ammo clamped to salvo size, no relaods
-        [BDAPersistentSettingsField] public static bool INFINITE_FUEL = false;              //Infinite propellant
-        [BDAPersistentSettingsField] public static bool INFINITE_EC = false;                          //Infinite electric charge
+        [BDAPersistentSettingsField(RWPFilter = true)] public static bool INFINITE_AMMO = false;              //infinite Bullets/rockets/laserpower
```

Comment:
I like the idea of using the `RWPFilter` field to define what should be stored or not, but not having to specify it for each field that might be used by RWP.
Instead, I think we can dynamically set `RWPFilter` to `true` or `false` for the relevant fields depending on what's read in from the `RWP_settings.cfg` file and which RWP round is currently selected (or switched from->to).